tools/test/audio/comp_run.sh

```diff
@@ -152,11 +160,11 @@ else
     VALGRIND_CMD=
 fi
 
-export LD_LIBRARY_PATH=$HOST_LIB:$TPLG_LIB
-
 # Run test bench
 echo "Command:         $HOST_EXE"
 echo "Argument:        $ARG"
-echo "LD_LIBRARY_PATH=${LD_LIBRARY_PATH}"
+if [[ -z $XTRUN ]]; then
+    echo "LD_LIBRARY_PATH=${LD_LIBRARY_PATH}"
```

Comment:
```suggestion
declare -p LD_LIBRARY_PATH || true
```

Reply:
I realized that all the LD_LIBRARY_PATH handling is unnecessary with static testbench version. This and few other lines will be deleted.

`declare -p` is interesting, I didn't know it.